code/game/objects/items/toys.dm

```diff
@@ -989,6 +988,65 @@
 	card_attack_verb = list("attacked", "sliced", "diced", "slashed", "cut")
 	resistance_flags = NONE
 
+/*
+ * YOU HAVE UNO IT CAME FREE WITH YOUR FUCKING PDA
+ */
+
+/obj/item/toy/cards/deck/uno
+	name = "deck of UNO cards"
+	desc = "A deck of space-grade UNO cards."
+	icon = 'icons/obj/toy.dmi'
+	deckstyle = "uno"
+	icon_state = "deck_uno_full"
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/toy/cards/deck/uno/Initialize()
+	. = ..()
+	populate_uno_deck()
+
+/obj/item/toy/cards/deck/uno/proc/populate_uno_deck() //RED GREEN YELLOW BLUE
+	icon_state = "deck_[deckstyle]_full"
+	cards = list()
+	for(var/i in 0 to 9)
```

Comment:
Instead of doing this, just reuse the populate_deck proc but dont call parent
```suggestion
/obj/item/toy/cards/deck/uno/populate_deck() //RED GREEN YELLOW BLUE
	icon_state = "deck_[deckstyle]_full"
	cards = list()
	for(var/i in 0 to 9)
```